src/components/select/select.js

```diff
@@ -648,7 +648,12 @@ function SelectMenuDirective($parse, $mdUtil, $mdTheming) {
         var mapFn;
 
         if (mode == 'html') {
-          mapFn = function(el) { return el.innerHTML; };
+          mapFn = function(el) {
```

Comment:
I don't quite follow what this map function is doing- could you add a bit more explanation?